src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue21980.cs

```diff
@@ -0,0 +1,25 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue21980 : _IssuesUITest
+	{
+		public override string Issue => "IndicatorView with DataTemplate does not render correctly";
+		
+		public Issue21980(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		[Test]
+		[Category(UITestCategories.IndicatorView)]
+		public void IndicatorViewShouldRenderCorrectly()
```

Comment:
I am going to trigger a build of the test on Windows again. But, could be crashing.

From the test output:
`The app was expected to be running still, investigate as possible crash`

Reply:
Hi! Unfortunately, I have no way of investigating it on Windows. Could you maybe look at this, if this PR makes sense in general?

Reply:
@MartyIX Thanks!

Reply:
@jsuarezruiz that would be great! I have no Windows machine